blog/2015/11/swdcretreat2015-university_course.html

```diff
@@ -0,0 +1,428 @@
+  In the second week, they are asked to do a statistical analysis and in a markdown cell explain why they chose that test and what the test means.
+  It's interesting to see them tell each other to use markdown cells in the peer code review.
+  They push each other to do it.
+  Their final presentation is in a Jupyter Notebook so they are well on their way to doing literate programming,
+  but I do not give an explicit class on it.
+</p>
+
+<p>
+  [J] I teach <a href='http://rmarkdown.rstudio.com/'>rmarkddown</a> on Day 1.
+  It enables them to make a plot which gets them excited.
+  <em>Then</em> you can make a web page.
+  <em>Then</em> you can easily put it on Github.
+  It makes it extremely rewarding to the students.
+  The peer review is helpful because they can provide feedback and have constant interaction between people.
+  The students discuss the layout of the repository, how easy was it to find scripts and understand what the code is doing, etc.
+</p>
+
+
+<h4>Grading code and technical work is difficult, how do you do it?</h4>
+
+<p>
+[J] One of other reasons of doing peer review is to get feedback to the students faster than if only the instructor is grading.
+</p>
+
+<h4>Jenny since you seem to have been doing this for a few years, do you have any available resources to share?</h4>
+
+<p>
+  [J] My writing on the teaching of the course has always been talks.
+  My Courses are online, and I have speaker decks.
+  <ul>
+    <li><a href='http://stat545-ubc.github.io/'>Data wrangling, exploration, and analysis with R</a></li>
+    <li><a href='http://stat540-ubc.github.io/'>Statistical Methods for High Dimensional Biology</a></li>
+  </ul>
+  There's also Ethan White's courses <a href='http://www.datacarpentry.org/semester-biology/syllabus/'>here</a> and <a href='http://www.programmingforbiologists.org/'>here</a>.
+<p>
+
+
+<h4>Software and Stats for biology is such a fast paced field, how do you even decide what to teach?</h4>
+
+<p>
+  [J] The 2 courses I've taught the past decade is "Data Analysis" where I am chasing technology.
+  It's exhausting, but I also want to learn it myself.
+  <br>
+  I currently do the opposite in my genomics class.
+  I started by chasing the technology for years, but every year almost required a completely new course re-write.
+  I eventually generalized it to the key genomics pipelines and statistical analysis.
+  For example, multiple comparison and empirical bayes.
+  I try not to chase specific sequencing platforms.
+  You can only be chasing technology a few places at a time.
+  <br>
+  Research wise, I've only been able to put these classes after I've gotten tenure.
+  This allmws me to put more effort into developing the course with fewer outside distractions.
+  I would say what I'm doing would not mesh well with being super productive research-wise
+  ***Jeff Leek***, Roger Peng, rafa irizarry, are doing on line courses and super productive research-wise.
```

Comment:
On Mon, Dec 07, 2015 at 09:11:52PM -0800, Daniel Chen wrote:
> got marked as out of date for some reason.

When the line referenced by a inline comment changes, GitHub shifts
that comment thread to a “Show outdated diff” block.  One of the
reasons I prefer email-based workflows [1,2] ;).

[1]: http://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#Public-Project-over-Email
[2]: http://git-scm.com/book/en/v2/Git-Commands-Email